notifier.cpp

```diff
@@ -98,7 +101,14 @@ int main(int argc, char** argv) {
         // }
         // if (s == NATS_OK) {
         // s = stanConnection_PublishAsync(sc, subj, pubMsg->payload, pubMsg->size, _pubAckHandler, (void*)pubMsg);
-        s = stanConnection_Publish(sc, subj, line.c_str(), line.size());
+        for (int i = 0; i < 24 * 1000; ++i) {
```

Comment:
is it ok that we can try to publish 1 message for 20 minutes?

Reply:
not ok. 
but also not ok, that client can't publish message in 30 seconds.